cmd/fyne_demo/screens/container.go

```diff
@@ -0,0 +1,109 @@
+package screens
+
+import (
+	"fmt"
+	"image/color"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/theme"
+	"fyne.io/fyne/widget"
+)
+
+// ContainerScreen loads a tab panel for containers and layouts
+func ContainerScreen() fyne.CanvasObject {
+	return widget.NewTabContainer(
+		widget.NewTabItem("Split", makeSplitTab()),
+		widget.NewTabItem("Scroll", makeScrollTab()),
+		// layouts
+		widget.NewTabItem("Border", makeBorderLayout()),
+		widget.NewTabItem("Box", makeBoxLayout()),
+		widget.NewTabItem("Grid", makeGridLayout()),
```

Comment:
Center, Fixed Grid, and Max layouts are missing here. (Form Layout too, but Form is shown in widget, so I think it's ok)

Reply:
I removed the FixedGrid because it is going away/deprecated and confusing.
Unfortunately max and center look almost the same. I think center is a good shout though. Max is less use externally.